lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -595,4 +598,76 @@
         }
         return o;
     });
+
+    platform.registerFunction(platform.FunctionKind.Math_min, function (value1, value2, ...values) {
+        // #sec-math.min
+
+        // If no arguments are given, the result is positive infinity
+        // If any value is NaN, the result is NaN. 
+        // The comparison of values to determine the smallest value is done using the Abstract Relational Comparison algorithm except that +0 is considered to be larger than -0.
+        if (arguments.length === 0 ) {
+            return __chakraLibrary.positiveInfinity;
+        }
+
+        if (__chakraLibrary.isNaN(value1)) return NaN;
+        if (value1 === null) value1 = 0;
+
+        if (arguments.length === 1) {
+            return value1;
+        }
+
+        if (__chakraLibrary.isNaN(value2)) return NaN;
+        if (value2 === null) value2 = 0;
+
+        if (arguments.length === 2) {
+            if ((value1 < value2) || (value1 === value2 && value1 === 0 && 1/value1 < 1/value2)) return value1; // checks for -0 and +0
+            else return value2;
+        }
+
+        let min = value1;
+
+        for (let i = 1; i < arguments.length; i++) {
+            if (arguments[i] === null) arguments[i] = 0;
+            if (__chakraLibrary.isNaN(arguments[i])) return NaN;
+            else if ((min > arguments[i]) || (min === arguments[i] && min === 0 && 1/min > 1/arguments[i])) min = arguments[i];
+        }
+        
+        return min;
+    });
+
+    platform.registerFunction(platform.FunctionKind.Math_max, function (value1, value2, ...values) {
+        // #sec-math.max
+
+        // If no arguments are given, the result is negative infinity
+        // If any value is NaN, the result is NaN. 
+        // The comparison of values to determine the largest value is done using the Abstract Relational Comparison algorithm except that +0 is considered to be larger than -0.
+        if (arguments.length === 0) {
+            return __chakraLibrary.negativeInfinity;
+        }
+        
+        if (__chakraLibrary.isNaN(value1)) return NaN;
+        if (value1 === null) value1 = 0;
+
+        if (arguments.length === 1) {
+            return value1;
+        }
+
+        if (__chakraLibrary.isNaN(value2)) return NaN;
+        if (value2 === null) value2 = 0;
+
+        if (arguments.length === 2) {
+            if ((value1 > value2) || (value1 === value2 && value1 === 0 && 1/value1 > 1/value2)) return value1;
+            else return value2;
+        }
+
+        let max = value1;
+
+        for (let i = 1; i < arguments.length; i++) {
+            if (arguments[i] === null) arguments[i] = 0;
+            if (__chakraLibrary.isNaN(arguments[i])) return NaN;
```

Comment:
Just a note: you are doing early exit here, but the spec says to convert all arguments to Number... 
This is an edge case, or course.